tests/utils.py

```diff
@@ -204,26 +204,48 @@ def __init__(self, caplog, index=0, allow_errors=True, ignore=[]):
         self.allow_errors = allow_errors
         self.ignore = ignore
 
-    def _find(self, loggerpart, level, msg, after=0):
+    def _find(self, loggerpart, level, msg, after=0, min_level: int = 0):
```

Comment:
This is a non-backwards compatible change. Is that intentional? The original implementation required the given log level to match the log level of a record. The new implementation will consider any log level by default, independent of the value of `level`.

Reply:
Or do you mean `and` instead of `or`?

Reply:
oh man, I managed to mess it up again.